### DISCUSSION TITLE
Domain whitelisting

### DISCUSSION THREAD

**Discussion by gokulraghuthaman** [ONDC-Official/developer-docs#70] [Q&A]
Hi,
I need to whitelist the domain for the environment access, so for that I am confused which domain should be whitelisted.
for example The frontend is hosted in abc.com and the backend is in backend.abc.com which one should be whitelisted?

**Comment by DivyakantSwarnkar01** [comment]
My website is not getting whitelisted why?

**Reply by dev-bunk** [reply]
Did you host the domain with an SSL certificate?

**Comment by DivyakantSwarnkar01** [comment]
Yes, I did everything as mentioned in document 📃

**Reply by DivyakantSwarnkar01** [reply]
Yes I have done all thing as said in Docs, They are not even answering in raised ticket?

**Comment by DivyakantSwarnkar01** [comment]
I have send POST Request to Staging Subscriber Registry with all details of my firm. But Some of these details are not appropriate or written wrong ways. I want to change the details on Registry of subscription in staging phase. What is the URL for sending ""UPDATE" to these registry?

https://staging.ondc.org/subscribe is for subscription but what is for updating registry details???

I tried ""/update"" and ""subscribe/update"" but not working either. Kindly send me reply to ==> myekkhoj@gmail.com or +917408765402
